R/timeseriessummary-data.R

```diff
@@ -583,7 +583,7 @@ unNestCorrectionParameters <- function(corrections, timezone) {
                                    "AdjustableTrim" = formatCorrectionsParamAdjustableTrim(UpperThresholdPoints, LowerThresholdPoints, timezone),
                                    "FillGaps" = formatCorrectionsParamFillGaps(ResamplePeriod, GapLimit),
                                    "Deviation" = formatCorrectionsParamDeviation(DeviationValue, DeviationType, WindowSizeInMinutes),
-                                   type)) %>% 
+                                   " ")) %>% 
```

Comment:
Does this never print the type then?

Reply:
This ticket requests that we don't print anything in that column if it doesn't have any parameters associated. 
It was defaulting to print the type when there were no parameters for some of the correction types (DeleteRegion, CopyPaste, Freehand and Revert To Raw) but Laura/et al don't want anything there when there are no parameters. The type still prints in the CorrectionType column, previously it was duplicating like this screenshot
![image](https://user-images.githubusercontent.com/6920344/30712176-c49b6f28-9ed0-11e7-96d5-7f039f40d1e3.png)

Reply:
👍